draft-wussler-openpgp-pqc.md

```diff
@@ -1465,14 +1465,14 @@ context.
 
 The `fixedInfo` defined in {{kem-fixed-info}} binds the derived KEK to the
 chosen algorithm and communication parties. The algorithm ID identifies
-univocally the algorithm, the parameters for its instantiation, and the length
+unequivocally the algorithm, the parameters for its instantiation, and the length
 of all artifacts, including the derived key.
 
 This is in line with the Recommendation for ECC in section 5.5 of
 [SP800-56A]. Other fields included in the recommendation are not relevant
 for the OpenPGP protocol, since the sender is not required to have a key of
 their own, there are no pre-shared secrets, and all the other parameters are
-univocally defined by the algorithm ID.
+unequivocally defined by the algorithm ID.
```

Comment:
I think both words mean the same but I am fine with changing it.

Reply:
agree with @teythoon's  proposed change here.  As a native en_US speaker with a decent vocabulary, unequivocally is a much more common and well-understood word choice.  i can parse "univocally" and figure out what it means ("with one voice") but it's not anything that i'm used to reading.